graphql/src/main/java/com/scalar/db/graphql/schema/TableGraphQlModel.java

```diff
@@ -65,7 +64,6 @@ public TableGraphQlModel(String namespaceName, String tableName, TableMetadata t
     this.tableName = Objects.requireNonNull(tableName);
     this.tableMetadata = Objects.requireNonNull(tableMetadata);
 
-    this.transactionEnabled = ConsensusCommitUtils.isTransactionalTableMetadata(tableMetadata);
```

Comment:
So, if transaction directive is specified for non-transactional table, 
the GraphQL layer doesn't care about it and expect the underlining layer to throw an exception?

Reply:
Right, the GraphQL layer doesn't check if a table has transactional metadata.
It simply works as told by the `@transaction` directive. For example, if you try to put a record to a non-transactional table with `@transaction`, a `CommitException` ("preparing records failed") is thrown when committing.